.github/workflows/r_valgrind.yml

```diff
@@ -0,0 +1,30 @@
+name: R valgrind tests
+
+on:
+  pull_request_review_comment:
+    types: [created]
+  issue_comment:
+    types: [created]
```

Comment:
@StrikerRUS I just  added this. I learned from https://docs.github.com/en/free-pro-team@latest/rest/reference/pulls#review-comments  that the `issue_comment` event is created when you make a comment on a pull request that is not part of a review.

Triggering the job without this, as a PR owner, was annoying because GitHub will not let me add myself as a reviewer on my own PR, and the only way to create a "comment" review is to start a new comment thread.

Reply:
/gha run r-valgrind

Reply:
^ this non-review comment just triggered this build: https://github.com/microsoft/LightGBM/actions/runs/312971120

Reply:
I think with `issue_comment` it will be quite hard to track what PR comment belongs. I had no problem with my own PR by just adding comment from `Files changed` tab. Could you please double check that single inline comment triggers CI and remove `on issue_comment` if it does?

![image](https://user-images.githubusercontent.com/25141164/96354991-aa8f3800-10e5-11eb-8d5b-3a42c83dd4f5.png)

Reply:
In addition, this pollutes Actions logs with skipped runs because new run is created with every new comment:

![image](https://user-images.githubusercontent.com/25141164/96355090-7c5e2800-10e6-11eb-8e17-4834edb097f0.png)

Reply:
the thing that is annoying is that for `pull_request_review_comment`, you have to create a comment in a new place in the diff every time.

Notice that my response here doesn't give me a "start a review" option

![image](https://user-images.githubusercontent.com/7608904/96355173-531ca800-10a4-11eb-8d49-3edf34c6462f.png)

and neither does the comment at the bottom:

![image](https://user-images.githubusercontent.com/7608904/96355183-692a6880-10a4-11eb-8c23-122c2a376043.png)

Although now that I say that, I guess I can avoid needing to leave a comment in a new place by deleting an old comment and adding new ones in the same place. It still feels very clunky but I guess it's ok. I didn't realize that all comments that don't trigger it would lead to a "skipped" message like in your second image above.

I'll remove this change

Reply:
ok yes I see now, you're right: https://github.com/microsoft/LightGBM/pull/3443/files#r506994205

Reply:
I'm able to trigger tests by my own comments in one  "review" thread:

![image](https://user-images.githubusercontent.com/25141164/96355368-8e8d9580-10e9-11eb-9f68-0c23922f7205.png)


Both those comments triggered CI.

> notice that this comment I just left triggered a run that was skipped, because it wasn't part of a review:

I guess you just had a typo with excess `-` sign, no magic 😄 .

My theory is that any correct (in terms of key words) comment in **review thread" can trigger CI, no matter from `Conversation` or `Files changed` tab it was left. Of course, review thread can be **started** only from `Files changed` tab.

Reply:
haha oh ok, my mistake! I think it is ok to leave it as-is then, no need to add `issue_comment`